v2.1/admin-ui-cluster-overview-page.md

```diff
@@ -13,9 +13,9 @@ The **Cluster Overview** panel provides the following metrics:
 
 Metric | Description
 --------|----
-Capacity Usage | <ul><li>The storage capacity used as a percentage of total storage capacity allocated across all nodes.</li><li>The current capacity usage.</li></ul>
-Node Status | <ul><li>The number of [live nodes](#live-nodes) in the cluster.</li><li>The number of suspect nodes in the cluster. A node is considered a suspect node if it's liveness status is unavailable or the node is in the process of decommissioning.</li><li>The number of [dead nodes](#dead-nodes) in the cluster.</li>
-Replication Status | <ul><li>The total number of ranges in the cluster.</li><li>The number of [under-replicated ranges](admin-ui-replication-dashboard.html#review-of-cockroachdb-terminology) in the cluster. A non-zero number indicates an unstable cluster.</li><li>The number of [unavailable ranges](admin-ui-replication-dashboard.html#review-of-cockroachdb-terminology) in the cluster. A non-zero number indicates an unstable cluster.</li>
+Capacity Usage | <ul><li>Used capacity: The storage capacity used by CockroachDB (represented as a percentage of total storage capacity allocated across all nodes).</li><li>Usable capacity: The sum of the storage capacity used by CockroachDB as well as capacity available for use by CockroachDB. </li></ul>
```

Comment:
> Usable capacity: The sum of the storage capacity used by CockroachDB as well as capacity available for use by CockroachDB

Using this for reference (I think we should have a diagram like this in the docs, since it's the only way I can remember this stuff): https://github.com/cockroachdb/cockroach/blob/3afbd217567aa86c847e9104bd928b871c295e12/pkg/storage/metrics.go#L272-L280

This might be better phrased as "the space available for data storage, i.e. the storage capacity of the machine minus the capacity used by the Cockroach binary, operating system, and other system files"